ReactNativeClient/lib/components/side-menu-content.js

```diff
@@ -148,8 +148,10 @@ class SideMenuContentComponent extends Component {
 						Alert.alert('', _('Delete notebook "%s"?\n\nAll notes and sub-notebooks within this notebook will also be deleted.', folder.title), [
 							{
 								text: _('OK'),
-								onPress: () => {
-									Folder.delete(folder.id);
+								onPress: async () => {
+									await Folder.delete(folder.id);
+									if (this.props.folders.length==0)
+										this.allNotesButton_press();
```

Comment:
What if the last notebook is deleted via synchronisation? Also I believe this happens not just when the last notebook is deleted but when the currently opened notebook is deleted (eg. via sync). This is a tricky problem that needs more work.

Reply:
Oh, I got your point, sir, I didn't check it with sync. Let me do the manual testing with sync and I will let you know. I tested it on mobile, this is the case only when the last notebook was getting deleted, not any notebook. It might be the case with sync. Can you suggest what might the approach to fix this, or should I try on my own first?

Reply:
As you suspected, I checked with sync and its not working with sync. But this is only the case with last notebook, all the notebooks when gets deleted,  the page of next notebook gets open, even in sync. So I will try to fix sync part of this also. I am trying to understand sync part of code right now and will get back to you as soon as I fix this.

Reply:
`if (!this.cancelling()) {
					for (let i = 0; i < localFoldersToDelete.length; i++) {
						const item = localFoldersToDelete[i];
						const noteIds = await Folder.noteIds(item.id);
						if (noteIds.length) {
							// CONFLICT
							await Folder.markNotesAsConflict(item.id);
						}
						await Folder.delete(item.id, { deleteChildren: false, changeSource: ItemChange.SOURCE_SYNC, trackDeleted: false });
					}
				}`

These are the code in syncronizer.js lines starting from 787. 
Sir, if I dispatch an action here, which will check if all the folders has been deleted, then it will redirect to home page. I think this should work as all the other test cases are getting passed. What do you think? And sir, I don't know why, sync is not working in emulators for me, but working in released apps and desktop clients. So, there is no way for me to check sync after changing something in my code. Can you please suggest something.

Reply:
@ishantgupta777 please take note of the rules for PR titles. Generally they should start with "Desktop:" or "Mobile:".  Please fix your titles.

@tessus could we update the instructions to include extra prefixes like All, Test, Docs?